src/utilities/permissions.ts

```diff
@@ -1045,6 +1099,9 @@ const queryPermissions: Record<GQLKeys, (user: User | null, ...args: any[]) => b
         (isPlanOwner(user, plan) || isPlanCollaborator(user, plan)))
     );
   },
+  UPDATE_SEEN_SOURCE_ENTRY: (user: User | null): boolean => {
```

Comment:
Should this not require the user to be plan owner/collaborator too?

Reply:
The user should also be prevented from even making the request via `FeaturePermissions`

Reply:
This query is now gone, but added the permissions to the new query, `UPDATE_DERIVATION_GROUP_ACKNOWLEDGED`, in [this commit](https://github.com/NASA-AMMOS/aerie-ui/pull/1396/commits/e6202284f201048906d0173af9f4cd6d91a8a23a)!